### ISSUE TITLE
Mapbox not showing in recordings

### ISSUE THREAD

**Issue by Jamiecharris** [microsoft/clarity#317]
Hi,

My recordings are not showing the mapbox map

**Comment by ClaritySupport** [comment]
@Jamiecharris , We are Clarity support team. Could you please share below details to our support alias ClarityMS@microsoft.com to investigate the issue further.

1. Clarity Project ID
2. Website address where Clarity script installed
3. Link to recording having the issue.
4. Screenshot describing the issue.

**Comment by nitrag** [comment]
It doesn't help if you don't post resolution here.

**Comment by fancyham** [comment]
@ClaritySupport Hi, is there a solution for this?  We have the same problem (see the issue mention above)

**Comment by ClaritySupport** [comment]
Hi,

Could you please log out and log in again and check if still you are facing the same issue please share the latest recording link so that we can further investigate.

Thanks,
Clarity Support Team
Visit our docs!: https://docs.microsoft.com/en-us/clarity
***@***.******@***.***>
***@***.***

Confidentiality note: This e-mail, and any attachment to it, contains privileged and confidential information intended only for the use of the individual(s) or entity named in the e-mail.  If the reader of the e-mail is not the intended recipient, or the employee or agent responsible for delivering it to the intended recipient, you are hereby notified that reading it is strictly prohibited.  If you have received this e-mail in error, please immediately return it to the sender and delete it from your system.

**Comment by fancyham** [comment]
Yes, we’re having the same problem — logging in and out doesn’t make a difference.  (sorry for the delayed response)
